TRANSLATING.md

```diff
@@ -59,7 +59,7 @@ If there is no project for your language, you can start your own translation.
 Base your work on the second edition of the book, available [here](https://github.com/progit/progit2). To do so:
  1. Pick the correct [ISO 639 code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) for your language.
  1. Create a [GitHub organization](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/creating-a-new-organization-from-scratch), for example: `progit2-[your code]` on GitHub.
- 1. Create a project ``progit2``.
+ 1. Create a project "`progit2"`.
```

Comment:
```suggestion
 1. Create a project `progit2`.
```

Undo changes the migration script has made.

Reply:
I think the correct change is:

```diff
- 1. Create a project ``progit2``.
+ 1. Create a project "`progit2`".
```

Reply:
That would be right if this was a AsciiDoctor file, but we're in a Markdown file.

Your proposal doesn't render properly in Markdown:

```
Create a project "`progit2'"
```

## Rendering preview:

Left is source, right is Markdown preview:
![rendering_translating_markdown](https://user-images.githubusercontent.com/34918129/95023430-4b5d0c80-067d-11eb-965a-75be03709b5a.png)

Reply:
Oh, ah. It's markdown here… Let's keep it as it was before the patch.